tests/by-util/test_seq.rs

```diff
@@ -800,3 +800,9 @@ fn test_invalid_format() {
         .no_stdout()
         .stderr_contains("format '%g%g' has too many % directives");
 }
+
+#[test]
+#[ignore = "slow test - see issue #6182"]
+fn test_big_values() {
+    new_ucmd!().args(&["4e4000003", "4e4000003"]).succeeds();
```

Comment:
This test is incorrect, it should ensure there is a failure because GNU `seq` fails with:
```
$ seq 4e4000003 4e4000003
seq: invalid floating point argument: ‘4e4000003’
Try 'seq --help' for more information.
```